metadata-ingestion/tests/unit/test_bigquery_source.py

```diff
@@ -602,6 +602,12 @@ def test_get_table_and_shard_custom_shard_pattern(
         ("project.dataset.table", "project.dataset.table"),
         ("project.dataset.table_20231215", "project.dataset.table"),
         ("project.dataset.table@1624046611000", "project.dataset.table"),
+        ("project.dataset.table@-9600", "project.dataset.table"),
+        ("project.dataset.table@-3600000", "project.dataset.table"),
+        ("project.dataset.table@-3600000--1800000", "project.dataset.table"),
```

Comment:
What does this double dash even mean lol, just curious

Reply:
that query is @(time1-time2) where time is represented as negative delta from current time

https://cloud.google.com/bigquery/docs/table-decorators#range-examples